examples/tokenvm/cmd/token-cli/cmd/action.go

```diff
@@ -117,12 +119,39 @@ var createAssetCmd = &cobra.Command{
 			return err
 		}
 
+		promptSupply := promptui.Prompt{
```

Comment:
Please use `promptInt` for this: https://github.com/ava-labs/hypersdk/blob/d7e8d760f7768c45e3d927877616b5f75cae5640/examples/tokenvm/cmd/token-cli/cmd/utils.go#L141-L166

I think we should probably change this to `promptUint64` (if you want to do that too)

Reply:
shouldnt we then also change the return type of `PromptInt` into uint64? I have already prepared a commit for that  @patrick-ogrady

Reply:
done

Reply:
Yes, you are correct.